### PR TITLE
Decouple fetching layer from Redux, part 5

### PR DESCRIPTION
**Pull request by axelboc**
Diffractometer API

### CODE REVIEW

File: ui/src/actions/sampleview.js [hunk 1]
```diff
@@ -677,23 +669,9 @@ export function getMotorPosition(motor) {
...
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-    })
-      .then((response) => {
-        if (response.status >= 400) {
-          throw new Error('Server refused to get motor position');
-        }
```
Comment (axelboc) [inline ui/src/actions/sampleview.js:691]: To clarify, `wretch` already throws when receiving HTTP error codes.

File: ui/src/containers/SampleViewContainer.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ class SampleViewContainer extends Component {
         <PhaseInput
           phase={this.props.sampleViewState.currentPhase}
           phaseList={this.props.sampleViewState.phaseList}
-          sendPhase={this.props.sampleViewActions.sendCurrentPhase}
+          sendPhase={this.props.sampleViewActions.changeCurrentPhase}
```
Comment (marcus-oscarsson) [inline ui/src/containers/SampleViewContainer.js]: What do you think about changing `sendPhase` to `changePhase` as well  ?

Comment (axelboc) [reply ui/src/containers/SampleViewContainer.js]: I noticed it when I worked on #1136  :wink:

Comment (marcus-oscarsson) [reply ui/src/containers/SampleViewContainer.js]: Oh yes, I noticed just now :)

**Pull request merged**